scripts/libs/utils.js

```diff
@@ -79,6 +79,33 @@ function getTestnetChains(chains = []) {
     }));
 }
 
+/**
+ * Get chains config for testnet.
+ * @param {*} chains - The list of chains to get the chain objects for. If this is empty, the default chains will be used.
+ * @returns {Chain[]} - The chain objects.
+ */
+function getMainnetChains(chains = []) {
+    const _path = path.join(__dirname, '../../chain-config/mainnet-evm.json');
```

Comment:
where does this file come from? why don't we just import the info files the axelar-chain-configs package published in deployments repo?

Reply:
It creates its own local config file and writes info for contracts deployed by examples.